lib/ansible/modules/windows/win_copy.py

```diff
@@ -43,8 +43,16 @@
       with "/" or "\", or C(src) is a directory.
     - If C(src) and C(dest) are files and if the parent directory of C(dest)
       doesn't exist, then the task will fail.
-    required: yes
     type: path
+    required: yes
+  backup:
+    description:
```

Comment:
What's the behaviour when we are copying a folder, or multiple files? Looking at the code it seems like this only applies when copying a single file and when copying multiple files for going the traditional control -> remote but applies to multiple files when doing remote_src: True.

Reply:
I still need to investigate this. Check the behaviour of both python **copy** and powershell **win_copy**.